webpack/assets/javascripts/react_app/components/common/forms/Actions.test.js

```diff
@@ -10,7 +10,7 @@ describe('actions', () => {
 
     expect(toJson(wrapper)).toMatchSnapshot();
   });
-  it('buttons could be disabled', () => {
+  it('submit button could be disabled', () => {
```

Comment:
Maybe change to `'should show disabled buttons'` ? I think it's helpful if the English grammar is correct when using `it`.